tests/e2e/pages/homepage.js

```diff
@@ -8,6 +8,13 @@ class HomepagePage extends BasePage {
 
     this.title = $('header a');
     this.recent_pages_list = $('aside ul');
+
+    // For testing "search"
```

Comment:
move to "search" page

Reply:
You'll have to show me how.. I tried making a search.js for it, but the "feature" starts on "homepage" -> `When I visit the "homepage" page`. From there it navigates to the search page, but the orignal 'page' still applies to it, as far as the javascript is concerned.

Reply:
`Then the "search" page is displayed` and it should switch current page object

Reply:
Done!